assets/RNAseq_report.Rmd

```diff
@@ -1428,7 +1455,7 @@ log2FoldChange_quotient <- length(seq(log2FoldChange_min, log2FoldChange_max, 10
 
 pg <- ggplot(DE_all, aes(x=log2FoldChange, y=logpval, text=paste("Gene: ", gene_name, "<br>", "Log2FC: ", formatC(log2FoldChange, digits=2)))) +
         #this goes after alpha=0.5:
-        geom_point(alpha=0.5, aes(color=ifelse(abs(DE_all$log2FoldChange) > params$logFC_threshold &  DE_all$logpval > -log10(params$pval_threshold), "Differentially expressed genes", "Non-differentially expressed genes"))) +
+        geom_jitter(alpha=0.5, aes(color=ifelse(abs(DE_all$log2FoldChange) >= params$logFC_threshold &  DE_all$logpval >= -log10(params$pval_threshold), "Differentially expressed genes", "Non-differentially expressed genes"))) +
```

Comment:
Nice 🥳  Is that sufficient or do you also need to specify `width=0.2` or so to make the dots shift sideways?

Reply:
Added `width=0.2` but in many cases probably it will not change much because the plots are often completely crowded